src/classes/settings.py

```diff
@@ -39,6 +41,18 @@ class SettingStore(JsonDataStore):
     """ This class only allows setting pre-existing keys taken from default settings file, and merges user settings
     on load, assumes default OS dir."""
 
+    @unique
+    class pathType(Enum):
+        RECENT=0 # Return the last used path for the same action
+        PROJECT=1 # Return the current project path
+
+    @unique
+    class actions(Enum):
```

Comment:
action => actionType: what do you think about renaming this to clearly be a 'type'?